feature-tests/utils/create-session.ts

```diff
@@ -185,27 +189,17 @@ export async function postRequestHmrcKbvCriVc() {
       });
     });
   expect(postHmrcKbvVCEndpoint.statusCode).toEqual(Number(200));
+  console.log("VC Headers", postHmrcKbvVCEndpoint, undefined, 2);
```

Comment:
This is logging out the entire postHmrcKbvVCEndpoint object and not the response headers

`console.log("VC Headers", postHmrcKbvVCEndpoint.header, undefined, 2);`

Can you confirm the above change shows that the VC is being returned as content type application/json and not application/jwt?

Reply:
Thanks Seamus, I have updated this to show only the headers for the VC call, and to include validation around the content-type header being application/json